xformers/benchmarks/benchmark_triton_dropout.py

```diff
@@ -69,17 +91,21 @@ def triton_step(x):
                     results[key] = {}
 
                 # Record BW
-                bandwidth = to_gbs_fw(a, time)
+                bandwidth = to_gbs_fw(a, time, bias)
                 results[key][testcase.name] = f"{bandwidth:.1f}"
 
         pretty_print(results, title="\n --- Type: {} --- ".format(dtype), units="GB/s")
         pretty_plot(
             results,
-            title="Dropout-FW{}-{}".format("+BW" if backward else "", dtype),
+            title="Dropout-Bias-{}-FW{}-{}-Act: {}".format(
+                bias, "+BW" if backward else "", dtype, activation
+            ),
             units="GB/s",
             dash_key="pytorch",
         )
 
 
-for bw in [False, True]:
-    bench_dropout(bw)
+for activation in [Activation.GeLU, None]:
```

Comment:
Should others be benchmarked as well or its pretty similar for all of them?

Reply:
Gelu is kind of the worst case for activations and Triton (it uses the real exp function and not the fastmath equivalent), and "None" negates the fusion (well, we fuse with nothingness :D), so these are kind of hard goals to test truthfully. It's a little slow to test with everything, but could be extended on a ad-hoc basis if people would like ?

Reply:
Ah okay, that makes sense! Guess its pretty straightforward to modify and add an activation if someone needs to